sycl/doc/GetStartedGuide.md

```diff
@@ -31,82 +30,85 @@ OpenCL&trade; API to offload computations to accelerators.
   * Windows: `Visual Studio` version 15.7 preview 4 or later -
     https://visualstudio.microsoft.com/downloads/
 
-## Create SYCL workspace
+## Create DPC++ workspace
 
-Throughout this document `SYCL_HOME` denotes the path to the local directory
-created as SYCL workspace. It might be useful to create an environment variable
+Throughout this document `DPCPP_HOME` denotes the path to the local directory
```

Comment:
This is a typical example where `DPCPP_HOME` is a good name that makes sense since it is related to this implementation.

Reply:
Thanks )